src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import React from 'react';
+
+interface TodoInfoProps {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
```

Comment:
The `userId` prop is defined in the `TodoInfoProps` interface but is not used within the component. Consider removing it if it's unnecessary, or use it if it's intended for some functionality.